src/apidoc/js/toc_filter.js

```diff
@@ -57,7 +57,8 @@ $(function() {
       tocFilterUpdate();
       updateTocForSelection(); } });
 
-  selector = "#api_sub .pjax_enabled a:not(.external)";
+  // excluding pdf hrefs
+  selector = "#api_sub .pjax_enabled a:not(.external, [href$='.pdf'])";
```

Comment:
I reviewed this before and I think it's good, but if you ever end up adding another type, consider using another class rather than looking at the extension. (For instance, a class "binary" for PDFs and Word docs.)

Reply:
Thanks for the suggestion @dmcassel , will definitely make it more generalize for binary files in future releases.